{"_id":"01d729b0-ffbe-11ec-8278-c132177ae00a","postId":"2022_02_cancellation-1-overview-f578fde1-7dc4-3df5-88ec-107dca6b348e","postUri":"https://blog.stephencleary.com/2022/02/cancellation-1-overview.html","replyTo":"6f287750-b255-11ec-9070-750395a7028e","authorEmailEncrypted":"VQ/CuiPX4JZlaMvUhuuRqF2T+qsTr/LPWJy+U9UFWqv1NJRPzf57AvU9/5n/z6KtLdW62EfvYuuxOE0g0Jds9QSDYehRRZe98jiq+WHr297PW5rY2VHH0QZdDQaimDec7A+4jVdivOnhIy1/hsPo7/ADXKasWLKMUOz7tzqw2XiRE5c6vAY9iXsmd/u/J+0pKz9hYSRNaJ9LomfKulIRkSfmPzV9sL4PuOPLooVOrMnHjDo8NG/B3JJifs7gDbbpGdvfUCxQtWioh3pXbf0MSVkjDKk+5YuUSBhGjHR1d/3JZmmlMWcy6ur76O2qWdbZxNi+MAnJtalR4b619OxAWU4XdpHFGorrmyzNZGJKWeY5An3TE8KFPGMFBmwGehQNV3hXhg/K2g5gNbBg2DB6/sW4hzlNP3ppXtWLStreoQofL4PqPVKtb7XxLLVqhscTIYK3itodvXxzg7BnvDLMw09IA8JtBxuV84jgq6mvDbYoXAlyWkgyPNim0faUhvm23H/1ULgUmDeu+3PGETaa2ouLlDR6I6B6PBk4ABdurT8xPXfUk8lbQeVGjyWlYBfgKCUbRmm9O0H96H1Adf8/daECxN2L9g9W13SA1X9z5/gKQBc9A2G2Vm/CoBe1i9s+fvr+9vlFCLE+e7imVZmUf6v6lYsJlULRUid4qXDKt0c=","authorEmailMD5":"23448e78d187fba56d8bc263081ac5d8","authorName":"Stephen Cleary","authorUri":"https://stephencleary.com","message":"Yes, but again, 99% of the time, code *shouldn't* care what the `Status` is.\r\n\r\nThe *result* of the task is the `OperationCanceledException`; whether the task faulted or was cancelled should be inferred from that result. And the result of tasks is most cleanly acquired via `await`, not `Status` / `Exception` / etc.","date":"2022-07-09T19:33:28.272Z"}